app/backend/knots.js

```diff
@@ -96,17 +96,17 @@ const createMakeFile = (knot, name) =>
     const { usesReplication: usesReplication = true } =
       specImplementation || {};
     /* eslint-disable no-template-curly-in-string */
-    const fileContent = `SHELL=/bin/bash -o pipefail\n\nfullSync:${EOL}\t-\tdocker run -v "$(CURDIR)/tap:/app/tap/data" --interactive ${
-      knot.tap.image
-    } ${
+    const fileContent = `SHELL=/bin/bash -o pipefail\n\nfullSync:${EOL}\t-\tdocker run -v "$(CURDIR)/tap:/app/tap/data"${
+      knot.tap.name === 'tap-s3-csv' ? ' -v $(HOME)/.aws:/root/.aws' : ''
```

Comment:
I would prefer if we captured this in a single place as tap metadata (i.e. constants.js).
You may call this `dockerParameters`.

In other words, for all taps but S3, `dockerParameters` would be unspecified.
For S3, it would be `-v $(HOME)/.aws:/root/.aws`.

As a general rule, there should be no logic, anywhere, related to how a tap should be handled, that is not based on tap metadata.